app/scripts/models/reliers/oauth.js

```diff
@@ -30,6 +30,7 @@ define([
       // redirectUri is used by the oauth flow
       redirectUri: null,
       scope: null,
+      accessType: Constants.ACCESS_TYPE_ONLINE,
```

Comment:
I suggest defaulting this to `null` and omitting it if not explicitly requested, so that the oauth-server can be solely in charge of that the default behaviour is.